### PR TITLE
PTX-19249 Add delete StorageCluster after each test

### PR DESCRIPTION
**Pull request by nikolaypopov**
Add delete StorageCluster after each test

### CODE REVIEW

File: test/integration_test/basic_test.go [hunk 1]
```diff
@@ -325,16 +325,26 @@ func BasicInstallInCustomNamespace(tc *types.TestCase) func(*testing.T) {
 			err = ci_utils.ValidateObjectsAreTerminated([]runtime.Object{ns}, false)
 			require.NoError(t, err)
 		}
+
+		// Delete and validate StorageCluster deletion
+		ci_utils.UninstallAndValidateStorageCluster(cluster, t)
```
Comment (ggriffiths) [inline test/integration_test/basic_test.go:330]: Maybe do a BeforeEach/AfterEach here to ensure this happens?

Comment (ggriffiths) [reply test/integration_test/basic_test.go:330]: nevermind, discussed offline

Comment (nikolaypopov) [reply test/integration_test/basic_test.go:330]: I think this won't work with the methods we have, let's test `px-wipe` after each test for all the different combinations we have

File: test/integration_test/basic_test.go [hunk 2]
```diff
@@ -109,7 +109,7 @@ var testStorageClusterBasicCases = []types.TestCase{
 		TestFunc: BasicUpgradeOperator,
 	},
 	{
-		TestName:        "BasicInstall",
+		TestName:        "BasicInstallWithNodeAffinity",
```
Comment (jrivera-px) [inline test/integration_test/basic_test.go:112]: Why are we using node affinity now?

**Review by jrivera-px — APPROVED** [review]

File: test/integration_test/basic_test.go (hunk 2 above)
Comment (nikolaypopov) [reply test/integration_test/basic_test.go:112]: To make it more clear ^^

**Review by ggriffiths — APPROVED** [review]

**Pull request merged**